docs/contributor_guide/contributing_documentation.rst

```diff
@@ -0,0 +1,35 @@
+.. _contributing_documentation:
+
+Contributing documentation
+------------------------------
+
+Documentation is formatted in restructured text (ReST). To contribute, make sure to install sphinx and its
+add-ons by running :code:`python scripts/install_requirements.py --pinned False` in the repository
+root directory. You may also need to `install Pandoc <https://pandoc.org/installing.html>`_. You can contribute
+updates to existing documentation by navigating to the relevant part of the repository, and editing the restructured
+text files corresponding to your updates.
+
+To build the webpage run the following command from the repository root
```

Comment:
This is already documented: https://fairlearn.org/main/contributor_guide/development_process.html#building-the-website

Reply:
Merged it with the existing docs after discussing with @michaelamoako :-)